compiler-cli/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    static_vcruntime::metabuild();
```

Comment:
Can this be included only for release builds + on Windows?

Reply:
If you will checkout implementation of `metabuild()` function, it early returns on not release builds or builds without msvc:

```rs
pub fn metabuild() {
    if env::var("CARGO_MANIFEST_DIR").is_err() {
        panic!("`metabuild` must be called from a build script");
    }
	println!("cargo:rerun-if-env-changed=PROFILE");

	// Early exit if not msvc or release
	if env::var("CARGO_CFG_TARGET_ENV").as_deref() != Ok("msvc") || env::var("PROFILE").as_deref() != Ok("release") {
		return;
	}
    
    override_msvcrt_lib();

	// Disable conflicting libraries that aren't hard coded by Rust.
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:libvcruntimed.lib");
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:vcruntime.lib");
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:vcruntimed.lib");
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:libcmtd.lib");
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:msvcrt.lib");
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:msvcrtd.lib");
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:libucrt.lib");
	println!("cargo:rustc-link-arg=/NODEFAULTLIB:libucrtd.lib");
    // Set the libraries we want.
	println!("cargo:rustc-link-arg=/DEFAULTLIB:libcmt.lib");
	println!("cargo:rustc-link-arg=/DEFAULTLIB:libvcruntime.lib");
	println!("cargo:rustc-link-arg=/DEFAULTLIB:ucrt.lib");
}
```